Resources/Prototypes/temporary_folder/Spider_Queen.yml

```diff
@@ -0,0 +1,1482 @@
+        amount: 1
+      offset: 0, 1
+      speech: action-speech-spell-spider
+
+- type: entity
+  id: ActionSpawnMagicarpSpellSpiderGhost
+  name: spider-spawn-egg-ghost
+  noSpawn: true
+  components:
+  - type: WorldTargetAction
+    useDelay: 350 #КД
+    range: 1
+    itemIconStyle: BigAction
+    icon:
+      sprite: Objects/Misc/eggspider.rsi
+      state: icon
+    event: !type:WorldSpawnSpellEvent
+      prototypes:
+      - id: SpiderEggGhost
+        amount: 1
+      offset: 0, 1
+      speech: action-speech-spell-spider
+
+- type: entity
+  id: ActionSpawnMagicarpSpellSpiderShooter 
+  name: spider-spawn-egg-shooter
+  noSpawn: true
+  components:
+  - type: WorldTargetAction
+    useDelay: 380 
+    range: 1
+    itemIconStyle: BigAction
+    icon:
+      sprite: Objects/Misc/eggspider.rsi
+      state: icon
+    event: !type:WorldSpawnSpellEvent
+      prototypes:
+      - id: SpiderEggShooter
+        amount: 1
+      offset: 0, 1
+      speech: action-speech-spell-spider
+
+- type: entity
+  id: ActionSpawnMagicarpSpellSpiderCrystal 
+  name: spider-spawn-kristal
+  description: spider-spawn-kristal-desc
+  noSpawn: true
+  components:
+  - type: WorldTargetAction
+    useDelay: 30 
+    range: 1
+    itemIconStyle: BigAction
+    icon:
+      sprite: Structures/Decoration/crystal.rsi
+      state: crystal_grey
+    event: !type:WorldSpawnSpellEvent
+      prototypes:
+      - id: ShardCrystalBlueXeno
+        amount: 1
+      offset: 0, 1
+      speech: action-speech-spell-spider
+      
+- type: entity
+  id: ShardCrystalBlueXeno 
+  parent: ShardCrystalBlue
+  name: spider-spawn-crystal
+  description: spider-spawn-crystal-desc
+  noSpawn: true
+  components:  
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 5
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          MaterialWebSilk:
+            min: 0
+            max: 0
+      - !type:PlaySoundBehavior
+        sound:
+          collection: WoodDestroy  
+
+- type: entity
+  id: ActionSpawnMagicarpSpellSpiderLittle 
+  name: spider-spawn-smallspider
+  description: spider-spawn-smallspider-desc
+  noSpawn: true
+  components:
+  - type: WorldTargetAction
+    useDelay: 40 
+    range: 2
+    itemIconStyle: BigAction
+    icon:
+      sprite: Mobs/Animals/spider.rsi
+      state: viper_dead
+    event: !type:WorldSpawnSpellEvent
+      prototypes:
+      - id: MobSpiderSpaceLittle
+        amount: 3
+      offset: 0, 1
+      speech: action-speech-spell-spider
+
+- type: entity
+  id: ActionSpawnMagicarpSpellSpiderWallWeb 
+  name: spider-spawn-wall
+  noSpawn: true
+  components:
+  - type: WorldTargetAction
+    useDelay: 8 
+    range: 1.5
+    itemIconStyle: BigAction
+    icon:
+      sprite: Structures/Walls/web.rsi
+      state: full
+      useDelay: 30
+    event: !type:WorldSpawnSpellEvent
+      useDelay: 25
```

Comment:
![image](https://github.com/user-attachments/assets/37e49713-cec0-4e72-b505-b12bc3b363e5)

Reply:
Thank you for pointing out the errors.